File: BackAPI/SinapsisAPI/Model/Mensaje.cs
namespace sinapsis.Model
{
    public class Mensaje
    {
        public int idMensaje { get; set; }
        public int idCampania { get; set; }
        public short estadoEnvio { get; set; }
        public DateTime fechaHoraEnvio { get; set; }
        public string mensaje { get; set; }
        public short estado { get; set; }
    }
}

File: BackAPI/proyecto-node/models/mensaje.js
class Mensaje {
   constructor(idMensaje, idCampania, estadoEnvio, fechaHoraEnvio, mensaje, estado) {
     this.idMensaje = idMensaje;
     this.idCampania = idCampania;
     this.estadoEnvio = estadoEnvio;
     this.fechaHoraEnvio = fechaHoraEnvio;
     this.mensaje = mensaje;
     this.estado = estado;
   }
 }
 
 module.exports = Mensaje; 
